src/Microsoft.PackageManagement/Implementation/PackageManagementService.cs

```diff
@@ -1037,11 +1037,16 @@ private IMetaProvider GetMetaProviderObject(IHostApi request)
                 throw new ArgumentNullException("request");
             }
 
+            var providerAssemblies = Enumerable.Empty<string>();
+
+#if !CORECLR
             //looks like registry needs to be here for supporting .msi packages
-            var providerAssemblies = (_initialized ? Enumerable.Empty<string>() : _defaultProviders)
+             providerAssemblies = (_initialized ? Enumerable.Empty<string>() : _defaultProviders)
```

Comment:
The indentation of this line is not right, you got 1 more space before `providerAssemblies`